tests/search/components/TestSearchForm.js

```diff
@@ -7,24 +7,30 @@ const render = ReactTestUtils.renderIntoDocument;
 const findByTag = ReactTestUtils.findRenderedDOMComponentWithTag;
 
 describe('<SearchForm />', () => {
+  var onSearch;
```

Comment:
To me it seems like `let` is just `var` with less footguns. So we should always use `let` and `const`.